pkg/plugins/golang/v2/scaffolds/webhook.go

```diff
@@ -79,18 +64,28 @@ func (s *webhookScaffolder) newUniverse() *model.Universe {
 }
 
 func (s *webhookScaffolder) scaffold() error {
-	if s.conversion {
-		fmt.Println(`Webhook server has been set up for you.
-You need to implement the conversion.Hub and conversion.Convertible interfaces for your CRD types.`)
+	doConversion := s.resource.Webhooks.Conversion
+
+	if s.config.IsV3() {
```

Comment:
why we have isV3 un the v2/scaffiolds?

Reply:
Project version (3-alpha) != plugin version (v2).

Reply:
Do we really need it? are we not checking if the plugin is supported or not before this point?

Reply:
In v2, the webhook call has nothing to update from the stored resource, which is only saved on create api call currently. This keeps the same behavior. In v3 though, it has to update the `Resource.Webhooks` field.